test/Turn-test.js

```diff
@@ -4,15 +4,16 @@ const expect = chai.expect;
 const Turn = require('../src/Turn');
 const Card = require('../src/Card');
 
-// const card = new Card();
-// const turn = new Turn();
 
-// beforeEach(function() {
-//   card = new Card(1, 'What allows you to define a set of related information using key-value pairs?', ['object', 'array', 'function'], 'object');
-//   turn = new Turn('object', card);
-// });
 
 describe('Turn', function() {
+  let card;
+  let turn;
```

Comment:
I was using const and trying to reassign a variable but using const prevents that.